sql_challenge/data/continent_map.csv

```diff
@@ -0,0 +1,264 @@
+country_code,continent_code
+ABW,NA
+AFG,AS
+AGO,AF
+AIA,NA
+ALA,EU
+ALB,EU
+AND,EU
+ANT,NA
+ARE,AS
+ARG,SA
+ARM,EU
+ARM,AS
+ARM,AF
+ASM,OC
+ATA,AN
+ATF,AN
+ATG,NA
+AUS,OC
+AUT,EU
+AZE,EU
+AZE,AS
+BDI,AF
+BEL,EU
+BEN,AF
+BES,NA
+BFA,AF
+BGD,AS
+BGR,EU
+BHR,AS
+BHS,NA
+BIH,EU
+BLM,NA
+BLR,EU
+BLZ,NA
+BMU,NA
+BOL,SA
+BRA,SA
+BRB,NA
+BRN,AS
+BTN,AS
+BVT,AN
+BWA,AF
+CAF,AF
+CAN,NA
+CCK,AS
+CHE,EU
+CHL,SA
+CHN,AS
+CIV,AF
+CMR,AF
+COD,AF
+COG,AF
+COK,OC
+COL,SA
+COM,AF
+CPV,AF
+CRI,NA
+CUB,NA
+CUW,NA
+CXR,AS
+CYM,NA
+CYP,EU
+CYP,AS
+CZE,EU
+DEU,EU
+DJI,AF
+DMA,NA
+DNK,EU
+DOM,NA
+DZA,AF
+ECU,SA
+EGY,AF
+ERI,AF
+ESH,AF
+ESP,EU
+EST,EU
+ETH,AF
+FIN,EU
+FJI,OC
+FLK,SA
+FRA,EU
+FRO,EU
+FSM,OC
+GAB,AF
+GBR,EU
+GEO,EU
+GEO,AS
+GGY,EU
+GHA,AF
+GIB,EU
+GIN,AF
+GLP,NA
+GMB,AF
+GNB,AF
+GNQ,AF
+GRC,EU
+GRD,NA
+GRL,NA
+GTM,NA
+GUF,SA
+GUM,OC
+GUY,SA
+HKG,AS
+HMD,AN
+HND,NA
+HRV,EU
+HTI,NA
+HUN,EU
+IDN,AS
+IMN,EU
+IND,AS
+IOT,AS
+IRL,EU
+IRN,AS
+IRQ,AS
+ISL,EU
+ISR,AS
+ITA,EU
+JAM,NA
+JEY,EU
+JOR,AS
+JPN,AS
+KAZ,EU
+KAZ,AS
+KEN,AF
+KGZ,AS
+KHM,AS
+KIR,OC
+KNA,NA
+KOR,AS
+KWT,AS
+LAO,AS
+LBN,AS
+LBR,AF
+LBY,AF
+LCA,NA
+LIE,EU
+LKA,AS
+LSO,AF
+LTU,EU
+LUX,EU
+LVA,EU
+MAC,AS
+MAF,NA
+MAR,AF
+MCO,EU
+MDA,EU
+MDG,AF
+MDV,AS
+MEX,NA
+MHL,OC
+MKD,EU
+MLI,AF
+MLT,EU
+MMR,AS
+MNE,EU
+MNG,AS
+MNP,OC
+MOZ,AF
+MRT,AF
+MSR,NA
+MTQ,NA
+MUS,AF
+MWI,AF
+MYS,AS
+MYT,AF
+NAM,AF
+NCL,OC
+NER,AF
+NFK,OC
+NGA,AF
+NIC,NA
+NIU,OC
+NLD,EU
+NOR,EU
+NPL,AS
+NRU,OC
+,OC
```

Comment:
I believe these 4 countries are null which will need to be displayed as N/A

Reply:
correct